### ISSUE TITLE
FHI-aims: IndexError: list index out of range

### ISSUE THREAD

**Issue by ebirkan** [lucydot/effmass#38]
Running effmass for FHI-aims output gives me following error:
```
Traceback (most recent call last):
  File "/home/-----/.conda/envs/ase-env/bin/effmass", line 8, in <module>
    sys.exit(cli())
  File "/home/-----/.conda/envs/ase-env/lib/python3.6/site-packages/effmass/cli.py", line 83, in cli
    data = inputs.DataAims(pathname)
  File "/home/-----/.conda/envs/ase-env/lib/python3.6/site-packages/effmass/inputs.py", line 475, in __init__
    path_list.append(int(words[8]))
IndexError: list index out of range
```
How can I solve this problem?

**Comment by lucydot** [comment]
Hi @ebirkan which version of effmass are you using - and did you install via pip or clone effmass from Github?  

I believe that @MatthiasGolomb recently updated the version on the github Main branch so that it is compatible with recent changes to FHI-aims - @MatthiasGolomb do you know if your latest changes will fix the issue? If we think that this might fix the problem you are having then I will tag and upload to the python package index so you can pip update.

Lucy
If

**Comment by MatthiasGolomb** [comment]
This looks related to the kpath in the Brillouin zone - @ebirkan do you mind sharing your in- and output files?

**Comment by ebirkan** [comment]
Hi @lucydot , I installed effmass 2.0.0 via pip.

Hi @MatthiasGolomb you can find input and output files here 
[effmass.zip](https://github.com/lucydot/effmass/files/6500179/effmass.zip)

**Comment by MatthiasGolomb** [comment]
It looks like the output format has changed, not sure if in general or for the MD_light output level - it now outputs the bands with tabs instead of simple whitespace. I will push a fix now, but @ebirkan if you don't want to wait you can change the tabs in the `output band` line to simple whitespace and it should work.

**Comment by ebirkan** [comment]
Hi. Any update regarding this issue? I still get the same error and changing all band files look like a big issue for me as I work with many systems.

**Comment by MatthiasGolomb** [comment]
Have you downloaded the source code and installed it or are you working with the pip installed version still?

**Comment by ebirkan** [comment]
I uninstalled and installed it again with pip

**Comment by MatthiasGolomb** [comment]
Can you try cloning the repo and installing from source? I don't know whether Lucy has pushed the latest changes to pip yet

**Comment by lucydot** [comment]
Hi @ebirkan this problem has been fixed and is reflected in the latest version on pip. If you upgrade effmass (`pip install --upgrade effmass`) it should hopefully work. Please let us know if not.

**Comment by lucydot** [comment]
I am closing the issue, please re-open if your problem is not fixed with the upgrade to effmass 2.2.0
